front/src/Weekview.js

```diff
@@ -75,16 +83,19 @@ class Weekview extends Component {
 
     //Changes week number state to next one
     nextWeekClick = (e) => {
+
         this.setState({
             state:'loading'
         })
         e.preventDefault();
         let testi = moment(this.state.dayNro, "YYYYMMDD")
+
+        console.log("dayNro statessa: " + this.state.dayNro)
```

Comment:
Don't forget to comment out console.logs (this is fine because you will continue work here(?))